src/plugins/data/common/es_query/kuery/node_types/literal.ts

```diff
@@ -17,13 +17,17 @@
  * under the License.
  */
 
-export function buildNode(value) {
+import { LiteralTypeBuildNode } from './types';
+
+type LiteralNodeValue = null | boolean | number | string;
```

Comment:
nit: Could avoid duplicating this type here by pulling it from the existing build node interface: 
```ts
export function buildNode(value: LiteralTypeBuildNode['value']): LiteralTypeBuildNode {
```